.buildkite/pipeline.yml

```diff
@@ -114,5 +117,23 @@ steps:
       - export USER=$(id -u)
       - bazel --version
       - go version
-      - make bazel.unittest
+
+      - export XML_OUTPUT_FILE=test.xml
+      - export BAZEL_TESTLOG="$(bazel info 2>/dev/null | grep bazel-testlogs | tr ' ' '\t' | cut -f2)"
+      - mkdir -p junit/$BUILDKITE_COMMIT
+
+      # in case of failure export xml testlog
+      - make bazel.unittest || find "$$BAZEL_TESTLOG" -name '*.xml' -exec sh -c 'cp -v {} junit/$BUILDKITE_COMMIT/"$(echo {}|rev|cut -d/ -f1,2,3|rev|tr / -)"' ';' && exit 1)
       - make bazel.build
+
+    artifact_paths:
+      -  junit/$BUILDKITE_COMMIT/*.xml
+
+  - wait:
+    continue_on_failure: true
+  - label: reports-bigcors
+    agents:
+      queue: "bigcores"
+    plugins:
+      - junit-annotate#v1.7.0:
+          artifacts: junit/$BUILDKITE_COMMIT/*.xml
```

Comment:
This won't work
Each job is sent to a different agent and the repo is cleared at the start of the job

Reply:
My bad ^^ I didn't understand that it uses the artifacts of the other jobs